File: Api/Api_B84211_B87107/Entities/Vacunas.cs
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;
using System.Linq;
using System.Threading.Tasks;

namespace Api_B84211_B87107.Entities
{
    public class Vacunas
    {
        [Key]
        public int id { get; set; } 
        public int cedula { get; set; }
        public string vacuna { get; set; }
        public string descripcion { get; set; }
        public string  fecha_aplicacion{ get; set; }
        public string fecha_prox_dos { get; set; }

    }
}

File: Android/app/src/main/java/ucr/ac/cr/proyecto3_b87107_b84211/modelos/Vacunas.java
package ucr.ac.cr.proyecto3_b87107_b84211.modelos;

public class Vacunas {
    private int id;
    private int cedula;
    private String vacuna;
    private String descripcion;
    private String  fecha_aplicacion;
    private String fecha_prox_dos;

    public int getId() {
        return id;
    }

    public void setId(int id) {
        this.id = id;
    }

    public int getCedula() {
        return cedula;
    }

    public void setCedula(int cedula) {
        this.cedula = cedula;
    }

    public String getVacuna() {
        return vacuna;
    }

    public void setVacuna(String vacuna) {
        this.vacuna = vacuna;
    }

    public String getDescripcion() {
        return descripcion;
    }

    public void setDescripcion(String descripcion) {
        this.descripcion = descripcion;
    }

    public String getFecha_aplicacion() {
        return fecha_aplicacion;
    }

    public void setFecha_aplicacion(String fecha_aplicacion) {
        this.fecha_aplicacion = fecha_aplicacion;
    }

    public String getFecha_prox_dos() {
        return fecha_prox_dos;
    }

    public void setFecha_prox_dos(String fecha_prox_dos) {
        this.fecha_prox_dos = fecha_prox_dos;
    }
}
